Add-ons/Umbraco-Plumber/Notifications-and-Reminders/index.md

```diff
@@ -0,0 +1,118 @@
+| Instance      | WorkflowInstanceViewModel | The view model data for the current workflow. Best explored via Intellisense.                                    |
+
+The `HtmlEmailBase` contains the following fields:
+
+| Fields       | Data Type      | Description                                                                                         |
+|--------------|----------------|-----------------------------------------------------------------------------------------------------|
+| SiteUrl      | string         | The public URL of your site.                                                                        |
+| NodeName     | string         | The name of the node from the current workflow.                                                     |
+| Type         | string         | The workflow type including the scheduled date (if exists).                                         |
+| EmailType    | EmailType      | An enum value representing the current email type that relates directly to the workflow task type.  |
+| To           | EmailUserModel | The model defining the receiver of the email.                                                       |
+| Email        | string         | The user's email address or a group address (if a group email is being sent).                       |
+| Name         | Name           | The user's name.                                                                                    |
+| Language     | string         | The user's language.                                                                                |
+| Id           | int            | The user's ID or group ID (when sending to a group email address).                                  |
+| IsGroupEmail | bool           | Is the email being sent to a generic group email address?                                           |
+
+Umbraco Plumber provides **Settings** for determining who receives emails at which stages of a workflow. While these are set to default values during installation, it's recommended to update your Notifications Settings to better suit your installation needs. Emails can be sent to:
+
+- **All**: All the participants in all workflow stages (previous and current).
+- **Admin**: The admin user.
+- **Author**: The user who initiated the workflow.
+- **Group**: All members of the group assigned to the current task.
+
+:::note
+Duplicate users are removed from email notifications.
+:::
+
+By default, all emails are sent to the **Group**. This might not always be an ideal situation. For example: cancelled workflows would be best sent to the **Author** only, likewise with rejected.
+
+It might be useful to notify **All** the participants of completed workflows but even this may be excessive. Depending on your website, you can adjust the best configuration.
+
+## Config, Group, and Tasks Notifications
```

Comment:
Maybe 'Config, Group, and Tasks Events', to avoid possible confusion with Notifications in Umbraco. I'm replacing all the EventHandlers with Notifications for v11, which I'll need to document (dozens of notifications, compared to the handful of current event handlers)

Reply:
Ah ok. so I'll change it back to Events.